docker_test.go

```diff
@@ -194,10 +263,51 @@ func blockOnHold() {
 
 	<-c
 }
+func GetEvent(url string, method string) (string, error) {
+	client := &http.Client{}
+	req, err := http.NewRequest(method, url, nil)
+
+	if err != nil {
+		fmt.Println(err)
+		return "", err
+	}
+	req.Header.Add("Authorization", "Basic cnVkZGVyOnBhc3N3b3Jk")
+	res, err := client.Do(req)
+	if err != nil {
+		fmt.Println(err)
+		return "", err
+	}
+	defer res.Body.Close()
+
+	body, err := io.ReadAll(res.Body)
+	if err != nil {
+		fmt.Println(err)
+		return "", err
+	}
+	return string(body), err
+}
+
+func SendPixelEvents(writeKey string) {
+	// Send pixel/v1/page
+	log.Println("Sending pixel/v1/page Event")
+	url := fmt.Sprintf("http://localhost:%s/pixel/v1/page?writeKey=%s&anonymousId=identified_user_id", httpPort, writeKey)
+	method := "GET"
+	resBody, _ := GetEvent(url, method)
```

Comment:
error not handled, consider returning err or passing t *testing.T to this method so you can do t.Fail()